test/bigtable/row.js

```diff
@@ -0,0 +1,785 @@
+      var response = {};
+
+      row.parent.mutate = function(m, callback) {
+        callback(error, response);
+      };
+
+      row.deleteCells(columns, function(err, apiResponse) {
+        assert.strictEqual(err, error);
+        assert.strictEqual(response, apiResponse);
+        done();
+      });
+    });
+  });
+
+  describe('get', function() {
+    it('should provide the proper request options', function(done) {
+      row.parent.getRows = function(reqOpts) {
+        assert.strictEqual(reqOpts.key, ROW_ID);
+        assert.strictEqual(FakeMutation.parseColumnName.callCount, 0);
+        done();
+      };
+
+      row.get(assert.ifError);
+    });
+
+    it('should create a filter for a single column', function(done) {
+      var keys = [
+        'a:b'
+      ];
+
+      var expectedFilter = [{
+        family: 'a'
+      }, {
+        column: 'b'
+      }];
+
+      row.parent.getRows = function(reqOpts) {
+        assert.deepEqual(reqOpts.filter, expectedFilter);
+        assert.strictEqual(FakeMutation.parseColumnName.callCount, 1);
+        done();
+      };
+
+      row.get(keys, assert.ifError);
+    });
+
+    it('should create a filter for multiple columns', function(done) {
+      var keys = [
+        'a:b',
+        'c:d'
+      ];
+
+      var expectedFilter = [{
+        interleave: [
+          [{
+            family: 'a'
+          }, {
+            column: 'b'
+          }], [{
+            family: 'c'
+          }, {
+            column: 'd'
+          }]
+        ]
+      }];
+
+      row.parent.getRows = function(reqOpts) {
+        assert.deepEqual(reqOpts.filter, expectedFilter);
+        assert.strictEqual(FakeMutation.parseColumnName.callCount, 2);
+        done();
+      };
+
+      row.get(keys, assert.ifError);
+    });
+
+    it('should respect supplying only family names', function(done) {
+      var keys = [
+        'a'
+      ];
+
+      var expectedFilter = [{
+        family: 'a'
+      }];
+
+      row.parent.getRows = function(reqOpts) {
+        assert.deepEqual(reqOpts.filter, expectedFilter);
+        assert.strictEqual(FakeMutation.parseColumnName.callCount, 1);
+        done();
+      };
+
+      row.get(keys, assert.ifError);
+    });
+
+    it('should return an error to the callback', function(done) {
+      var error = new Error('err');
+      var response = {};
+
+      row.parent.getRows = function(r, callback) {
+        callback(error, null, response);
+      };
+
+      row.get(function(err, row_, apiResponse) {
+        assert.strictEqual(error, err);
+        assert.strictEqual(row_, null);
+        assert.strictEqual(response, apiResponse);
+        done();
+      });
+    });
+
+    it('should return a custom error if the row is not found', function(done) {
+      var response = {};
+
+      row.parent.getRows = function(r, callback) {
+        callback(null, [], response);
+      };
+
+      row.get(function(err, row_, apiResponse) {
+        assert(err instanceof Error);
```

Comment:
Export and test the `RowError` independently, then check here that the RowError was created and returned properly.